### ISSUE TITLE
Request for 1-Second Sampling Interval

### ISSUE THREAD

**Issue by tuhochi** [ehorvat1/NES-MEP-Reader#9]
Hi there,

First, thank you for this great project! It has been incredibly useful.

I would like to request an enhancement to the sampling interval. Currently, the data is fetched every 2 seconds, but I believe it would be more beneficial if the sampling could be done every 1 second instead. This change would provide more granular data, which is essential for my use case.

Thank you for considering this request. Keep up the excellent work!

Best regards, Hochi

**Comment by ehorvat1** [comment]
By the way: One can check the actual data read rate from MEP port by using Serial or Telnet debugging in "Info" level. Watch out for "---- Read MEP Port ----  actual millis: ....." which is prompted right after requesting to read MEP port.

**Comment by tuhochi** [comment]
Hello Bert!

I wanted to express my sincere thanks for the update and for providing the "firmware-1second.bin". I have successfully updated the firmware, and it seems like the sampling is now correctly set to one second. This is really great and very helpful for me!

Regarding the MAX3232 chip: I used the board design from Raffi. He mentioned that he had already sent the exact specifications earlier. After initial tests, it seems to be stable with the 1-second sampling time.

Once again, thank you very much for your quick support and the adjustments!

Best regards,
Hochi
